diff -Naur src.orig/etc/Makefile src/etc/Makefile
--- src.orig/etc/Makefile	2025-06-06 00:04:20.000000000 +0000
+++ src/etc/Makefile	2025-06-30 06:48:31.416442000 +0000
@@ -25,9 +25,6 @@
 	mailertable.sample
 .endif
 
-# Special top level files for FreeBSD
-FREEBSD=COPYRIGHT
-
 # Sanitize DESTDIR
 DESTDIR:=	${DESTDIR:C://*:/:g}
 
@@ -54,12 +51,8 @@
 	@echo "set DESTDIR before running \"make ${.TARGET}\""
 	@false
 .endif
-	${_+_}cd ${.CURDIR}/gss; ${MAKE} install
 	${_+_}cd ${.CURDIR}/mtree; ${MAKE} install
 	${_+_}cd ${SRCTOP}/share/termcap; ${MAKE} etc-termcap
-	${_+_}cd ${SRCTOP}/usr.sbin/rmt; ${MAKE} etc-rmt
-	${INSTALL_SYMLINK} -T "package=runtime" ../var/run/os-release \
-		${DESTDIR}/etc/os-release
 .if ${MK_UNBOUND} != "no"
 	if [ ! -e ${DESTDIR}/etc/unbound ]; then \
 		${INSTALL_SYMLINK} -T "package=unbound" \
@@ -89,8 +82,6 @@
 	cd ${.CURDIR}/mail; ${INSTALL} -o ${BINOWN} -g ${BINGRP} -m 644 \
 		-T "package=sendmail" ${ETCMAIL_SENDMAIL} ${DESTDIR}/etc/mail
 .endif
-	cd ${.CURDIR}/..; ${INSTALL} -o ${BINOWN} -g ${BINGRP} -m 444 \
-		-T "package=runtime" ${FREEBSD} ${DESTDIR}/
 .if ${MK_BOOT} != "no"
 .if exists(${SRCTOP}/sys/${MACHINE}/conf/GENERIC.hints)
 	${INSTALL} -o ${BINOWN} -g ${BINGRP} -m 444 \
@@ -103,8 +94,7 @@
 MTREES=		mtree/BSD.root.dist		/		\
 		mtree/BSD.var.dist		/var		\
 		mtree/BSD.usr.dist		/usr		\
-		mtree/BSD.include.dist		/usr/include	\
-		mtree/BSD.debug.dist		/usr/lib
+		mtree/BSD.include.dist		/usr/include
 .for LIBCOMPAT libcompat in ${_ALL_LIBCOMPATS_libcompats}
 .if ${MK_LIB${LIBCOMPAT}} != "no"
 MTREES+=	mtree/BSD.lib${libcompat}.dist	/usr
