diff -Naur src.orig/etc/mtree/BSD.include.dist src/etc/mtree/BSD.include.dist
--- src.orig/etc/mtree/BSD.include.dist	2025-11-28 04:15:53.000000000 +0400
+++ src/etc/mtree/BSD.include.dist	2026-01-06 13:49:28.928392000 +0400
@@ -6,14 +6,8 @@
 .
     arpa
     ..
-    atf-c               tags=package=atf-dev
-    ..
-    atf-c++             tags=package=atf-dev
-    ..
     bsm                 tags=package=audit-dev
     ..
-    bsnmp               tags=package=bsnmp-dev
-    ..
     c++
         v1
             __algorithm
@@ -149,8 +143,6 @@
         ..
         mfi
         ..
-        mlx5
-        ..
         mmc
         ..
         mpt
@@ -167,8 +159,6 @@
         ..
         pci
         ..
-        powermac_nvram
-        ..
         ppbus
         ..
         pwm
@@ -185,13 +175,9 @@
         ..
         vkbd
         ..
-        vmm
-        ..
         wg
         ..
     ..
-    devdctl             tags=package=utilities-dev
-    ..
     edit                tags=package=runtime-dev
         readline        tags=package=runtime-dev
         ..
@@ -252,36 +238,12 @@
         virstor
         ..
     ..
-    gssapi              tags=package=kerberos-dev
-    ..
-    gssrpc              tags=package=kerberos-dev
-    ..
-    infiniband          tags=package=utilities-dev
-        complib         tags=package=utilities-dev
-        ..
-        iba             tags=package=utilities-dev
-        ..
-        opensm          tags=package=utilities-dev
-        ..
-        vendor          tags=package=utilities-dev
-        ..
-    ..
     isofs
         cd9660
         ..
     ..
-    kadm5               tags=package=kerberos-dev
-    ..
-    krb5                tags=package=kerberos-dev
-    ..
     lib80211            tags=package=runtime-dev
     ..
-    lib9p               tags=package=lib9p-dev
-    ..
-    libipt
-    ..
-    libmilter           tags=package=libmilter-dev
-    ..
     libxo               tags=package=runtime-dev
     ..
     lzma                tags=package=xz-dev
@@ -299,10 +261,6 @@
     net80211
     ..
     netgraph
-        bluetooth
-            include
-            ..
-        ..
         netflow
         ..
     ..
@@ -332,50 +290,18 @@
     ..
     nfsclient
     ..
-    nfsserver
-    ..
-    opencsd
-        c_api
-        ..
-        ete
-        ..
-        etmv3
-        ..
-        etmv4
-        ..
-        ptm
-        ..
-        stm
-        ..
-    ..
     openssl             tags=package=openssl-dev
     ..
     pcap                tags=package=utilities-dev
     ..
     protocols
     ..
-    rdma                tags=package=utilities-dev
-    ..
     rpc
     ..
     rpcsvc
     ..
     security
         audit           tags=package=audit-dev
-        ..
-        mac_biba
-        ..
-        mac_bsdextended
-        ..
-        mac_grantbylabel
-        ..
-        mac_lomac
-        ..
-        mac_mls
-        ..
-        mac_partition
-        ..
-        mac_veriexec
         ..
     ..
     ssp
