diff -Naur src.orig/etc/Makefile src/etc/Makefile
--- src.orig/etc/Makefile	2026-01-08 19:00:02.184802000 +0400
+++ src/etc/Makefile	2026-01-10 15:14:14.567209000 +0400
@@ -23,9 +23,6 @@
 	mailertable.sample
 .endif
 
-# Special top level files for FreeBSD
-FREEBSD=COPYRIGHT
-
 # Sanitize DESTDIR
 DESTDIR:=	${DESTDIR:C://*:/:g}
 
@@ -53,16 +50,8 @@
 	@echo "set DESTDIR before running \"make ${.TARGET}\""
 	@false
 .endif
-.if ${MK_MITKRB5} != "no"
-	${_+_}cd ${.CURDIR}/gss-krb5; ${MAKE} install
-.else
-	${_+_}cd ${.CURDIR}/gss; ${MAKE} install
-.endif
 	${_+_}cd ${.CURDIR}/mtree; ${MAKE} install
 	${_+_}cd ${SRCTOP}/share/termcap; ${MAKE} etc-termcap
-	${_+_}cd ${SRCTOP}/usr.sbin/rmt; ${MAKE} etc-rmt
-	${INSTALL_SYMLINK} -T "package=runtime" ../var/run/os-release \
-		${DESTDIR}/etc/os-release
 .if ${MK_UNBOUND} != "no"
 # If NO_ROOT is defined, we are doing a stage install and always need to
 # install the /etc/unbound symlink, otherwise, don't overwrite the user's
@@ -99,8 +88,6 @@
 	cd ${.CURDIR}/mail; ${INSTALL} -o ${BINOWN} -g ${BINGRP} -m 644 \
 		-T "package=sendmail" ${ETCMAIL_SENDMAIL} ${DESTDIR}/etc/mail
 .endif
-	cd ${.CURDIR}/..; ${INSTALL} -o ${BINOWN} -g ${BINGRP} -m 444 \
-		-T "package=runtime" ${FREEBSD} ${DESTDIR}/
 .if ${MK_BOOT} != "no"
 .if exists(${SRCTOP}/sys/${MACHINE}/conf/GENERIC.hints)
 	${INSTALL} -o ${BINOWN} -g ${BINGRP} -m 444 \
@@ -113,8 +100,7 @@
 MTREES=		mtree/BSD.root.dist		/		\
 		mtree/BSD.var.dist		/var		\
 		mtree/BSD.usr.dist		/usr		\
-		mtree/BSD.include.dist		/usr/include	\
-		mtree/BSD.debug.dist		/usr/lib
+		mtree/BSD.include.dist		/usr/include
 .for LIBCOMPAT libcompat in ${_ALL_LIBCOMPATS_libcompats}
 .if ${MK_LIB${LIBCOMPAT}} != "no"
 MTREES+=	mtree/BSD.lib${libcompat}.dist	/usr
