diff -Naur src.orig/etc/mtree/BSD.root.dist src/etc/mtree/BSD.root.dist
--- src.orig/etc/mtree/BSD.root.dist	2024-11-29 03:03:41.000000000 +0300
+++ src/etc/mtree/BSD.root.dist	2025-06-01 16:27:16.688291000 +0300
@@ -9,16 +9,6 @@
     boot
         defaults
         ..
-        dtb
-            allwinner             tags=package=runtime
-            ..
-            overlays              tags=package=runtime
-            ..
-            rockchip              tags=package=runtime
-            ..
-        ..
-        efi
-        ..
         firmware
         ..
         loader.conf.d             tags=package=bootloader
@@ -29,8 +19,6 @@
         ..
         modules
         ..
-        uboot
-        ..
         zfs
         ..
     ..
@@ -39,58 +27,20 @@
     etc
         X11
         ..
-        authpf
-        ..
-        autofs
-        ..
-        bluetooth
-        ..
         cron.d
         ..
         defaults
         ..
         devd
         ..
-        dma
-        ..
-        gss
-        ..
-        jail.conf.d
-        ..
-        kyua             tags=package=tests
-        ..
-        mail
-        ..
         mtree
         ..
-        newsyslog.conf.d
-        ..
-        ntp             mode=0700
-        ..
         pam.d
         ..
-        periodic
-            daily
-            ..
-            monthly
-            ..
-            security
-            ..
-            weekly
-            ..
-        ..
         pkg
         ..
-        ppp
-        ..
-        profile.d
-        ..
-        rc.conf.d
-        ..
         rc.d
         ..
-        security
-        ..
         ssh
         ..
         ssl
@@ -99,13 +49,9 @@
             untrusted
             ..
         ..
-	sysctl.kld.d
-	..
         syslog.d
         ..
         zfs
-            compatibility.d
-            ..
         ..
     ..
     lib
@@ -118,15 +64,7 @@
         resolvconf
         ..
     ..
-    media
-    ..
     mnt
-    ..
-    net
-    ..
-    proc            mode=0555
-    ..
-    rescue
     ..
     root            mode=0750
     ..
