diff -Naur src.orig/include/Makefile src/include/Makefile
--- src.orig/include/Makefile	2025-06-06 00:04:20.000000000 +0000
+++ src/include/Makefile	2025-08-26 16:20:49.435381000 +0000
@@ -48,7 +48,7 @@
 	syslog.h ucontext.h
 
 LDIRS=	geom net net80211 netgraph netinet netinet6 \
-	netipsec netlink netsmb nfs nfsclient nfsserver sys vm
+	netipsec netlink netsmb nfs nfsclient sys vm
 
 LSUBDIRS=	dev/acpica dev/agp dev/ciss dev/filemon dev/firewire \
 	dev/hwpmc dev/hyperv \
@@ -67,9 +67,6 @@
 	netinet/netdump \
 	netinet/tcp_stacks \
 	netlink/route \
-	security/mac_biba security/mac_bsdextended security/mac_lomac \
-	security/mac_mls security/mac_partition \
-	security/mac_veriexec \
 	sys/disk \
 	ufs/ffs ufs/ufs
 
