diff -Naur src.orig/kerberos5/lib/Makefile src/kerberos5/lib/Makefile
--- src.orig/kerberos5/lib/Makefile	2025-06-05 20:04:20.000000000 -0400
+++ src/kerberos5/lib/Makefile	2025-09-12 05:57:59.157747000 -0400
@@ -1,17 +1,12 @@
 
 .include <src.opts.mk>
 
-SUBDIR=	libasn1 libhdb \
-	libheimntlm libhx509 libkadm5clnt libkadm5srv libkrb5 \
-	libroken libsl libvers libkdc libwind libheimbase libheimipcc libheimipcs
+SUBDIR=	libasn1 \
+	libheimntlm libhx509 libkrb5 \
+	libroken libvers libwind libheimbase libheimipcc
 
-SUBDIR+= libkafs5  # requires krb_err.h from libkrb5
-SUBDIR_DEPEND_libkafs5=	libkrb5
-
 .if ${MK_GSSAPI} != "no"
 SUBDIR+=	libgssapi_krb5
-SUBDIR+=	libgssapi_ntlm
-SUBDIR+=	libgssapi_spnego
 .endif
 
 .include <bsd.subdir.mk>
