diff -Naur src.orig/lib/libc/net/Makefile.inc src/lib/libc/net/Makefile.inc
--- src.orig/lib/libc/net/Makefile.inc	2025-06-06 00:04:20.000000000 +0000
+++ src/lib/libc/net/Makefile.inc	2025-09-02 09:45:56.387069000 +0000
@@ -3,7 +3,7 @@
 # machine-independent net sources
 .PATH: ${LIBC_SRCTOP}/net
 
-CONFS+=	net/hosts net/hosts.equiv net/networks net/nsswitch.conf net/protocols
+CONFS+=	net/hosts net/nsswitch.conf net/protocols
 SRCS+=	base64.c ether_addr.c eui64.c \
 	gai_strerror.c getaddrinfo.c \
 	gethostbydns.c gethostbyht.c gethostbynis.c gethostnamadr.c \
