diff -Naur src.orig/lib/Makefile src/lib/Makefile
--- src.orig/lib/Makefile	2026-01-08 19:00:00.835633000 +0400
+++ src/lib/Makefile	2026-01-10 18:28:31.321924000 +0400
@@ -29,17 +29,11 @@
 
 SUBDIR=	${SUBDIR_BOOTSTRAP} \
 	.WAIT \
-	libsqlite3 \
 	geom \
-	lib9p \
-	libalias \
 	libarchive \
-	libauditd \
-	libbegemot \
 	libblocksruntime \
 	libbsddialog \
 	libbsdstat \
-	libbsm \
 	libbz2 \
 	libcalendar \
 	libcam \
@@ -48,14 +42,12 @@
 	libcompat \
 	libcrypt \
 	libdevctl \
-	libdevdctl \
 	libdevinfo \
 	libdevstat \
 	libdl \
 	libdwarf \
 	libedit \
 	libelftc \
-	libevent1 \
 	libexecinfo \
 	libexpat \
 	libfetch \
@@ -64,7 +56,6 @@
 	libgeom \
 	libifconfig \
 	libipsec \
-	libiscsiutil \
 	libjail \
 	libkiconv \
 	libkldelf \
@@ -74,53 +65,32 @@
 	libmemstat \
 	libmd \
 	libmixer \
-	libmt \
 	lib80211 \
 	libnetbsd \
-	libnetmap \
 	libnv \
 	libnvmf \
 	libopenbsd \
 	libpam \
-	libpathconv \
 	libpcap \
-	libpjdlog \
-	libproc \
 	libprocstat \
 	libregex \
-	librpcsvc \
-	librss \
 	librt \
-	librtld_db \
-	libsamplerate \
 	libsbuf \
-	libsmb \
 	libstdbuf \
-	libstdthreads \
 	libsysdecode \
-	libtacplus \
 	libthr \
-	libthread_db \
 	libucl \
-	libufs \
-	libugidfw \
 	libulog \
 	libutil \
-	libutil++ \
-	${_libvgl} \
-	libwrap \
 	libxo \
 	liby \
-	libyaml \
 	libz \
 	libzstd \
 	ncurses \
-	nss_tacplus
 
 # Inter-library dependencies.  When the makefile for a library contains LDADD
 # libraries, those libraries should be listed as build order dependencies here.
 
-SUBDIR_DEPEND_geom=	libufs
 SUBDIR_DEPEND_googletest= libregex
 SUBDIR_DEPEND_libarchive= libz libbz2 libexpat liblzma libmd libzstd
 SUBDIR_DEPEND_libauditdm= libbsm
@@ -142,7 +112,7 @@
 .endif
 SUBDIR_DEPEND_libmagic= libz
 SUBDIR_DEPEND_libmemstat= libkvm
-SUBDIR_DEPEND_libpam= libcrypt ${_libradius} librpcsvc libtacplus libutil ${_libypclnt} ${_libcom_err}
+SUBDIR_DEPEND_libpam= libcrypt ${_libradius} libutil ${_libypclnt} ${_libcom_err}
 SUBDIR_DEPEND_libpjdlog= libutil
 SUBDIR_DEPEND_libprocstat= libkvm libutil
 SUBDIR_DEPEND_libradius= libmd
@@ -156,8 +126,6 @@
 .if ${MK_OFED} != "no"
 SUBDIR_DEPEND_libpcap= ofed
 .endif
-SUBDIR_DEPEND_nss_tacplus= libtacplus
-SUBDIR_DEPEND_virtual_oss= libsamplerate
 
 # NB: keep these sorted by MK_* knobs
 
@@ -175,7 +143,6 @@
 .endif
 
 SUBDIR.${MK_CUSE}+= 	libcuse
-SUBDIR.${MK_CUSE}+=	virtual_oss
 SUBDIR.${MK_TOOLCHAIN}+=libpe
 SUBDIR.${MK_DIALOG}+=	libdpv libfigpar
 SUBDIR.${MK_FDT}+=	libfdt
@@ -225,13 +192,6 @@
 SUBDIR.${MK_BHYVE}+=	libvmmapi
 .endif
 
-.if ${MACHINE_ARCH} != "powerpc" && ${MACHINE_CPUARCH} != "arm"
-SUBDIR.${MK_OPENMP}+=	libomp
-.endif
-.if ${MK_USB} != "no"
-SUBDIR.${MK_OPENSSH}+=	libcbor libfido2
-.endif
-SUBDIR.${MK_OPENSSL}+=	libmp
 SUBDIR.${MK_PF}+=	libpfctl
 SUBDIR.${MK_PMC}+=	libpmc libpmcstat
 SUBDIR.${MK_RADIUS_SUPPORT}+=	libradius
