diff -Naur src.orig/libexec/Makefile src/libexec/Makefile
--- src.orig/libexec/Makefile	2026-06-12 10:36:43.000000000 +0400
+++ src/libexec/Makefile	2026-06-14 21:59:50.850206000 +0400
@@ -16,16 +16,9 @@
 	${_phttpget} \
 	${_pppoed} \
 	rc \
-	revnetgroup \
 	${_rlogind} \
-	rpc.rquotad \
-	rpc.rstatd \
-	rpc.rusersd \
-	rpc.rwalld \
-	rpc.sprayd \
 	${_rshd} \
 	${_rtld-elf} \
-	save-entropy \
 	${_nuageinit} \
 	${_smrsh} \
 	${_tests} \
@@ -72,10 +65,6 @@
 
 .if ${MK_NETGRAPH} != "no"
 _pppoed=	pppoed
-.endif
-
-.if ${MK_PF} != "no"
-_tftp-proxy=	tftp-proxy
 .endif
 
 .if !defined(NO_PIC) && !defined(NO_RTLD)
