diff -Naur src.orig/libexec/rc/Makefile src/libexec/rc/Makefile
--- src.orig/libexec/rc/Makefile	2025-11-28 04:15:53.000000000 +0400
+++ src/libexec/rc/Makefile	2025-12-26 18:03:29.138442000 +0400
@@ -1,27 +1,18 @@
 .include <src.opts.mk>
 
-CONFGROUPS=	CONFETC CONFETCEXEC CONFETCDEFAULTS
+CONFGROUPS=	CONFETC CONFETCEXEC
 CONFETCDIR=	/etc
-CONFETC=	network.subr rc rc.initdiskless rc.subr rc.shutdown rc.bsdextended
+CONFETC=	rc.conf
 CONFETCPACKAGE=	rc
 
 .if ${MK_IPFW} != "no"
 CONFETC+=	rc.firewall
 .endif
 CONFETCMODE=	644
-CONFETCEXEC=	netstart pccard_ether rc.resume rc.suspend
+CONFETCEXEC=	rc rc.shutdown rc.resume rc.suspend
 CONFETCEXECDIR=	/etc
 CONFETCEXECMODE=	755
 CONFETCEXECPACKAGE=	rc
-CONFETCDEFAULTSDIR=	/etc/defaults
-CONFETCDEFAULTS=	rc.conf
-CONFETCDEFAULTSPACKAGE=	rc
-
-FILESGROUPS=	LIBEXEC_SCRIPTS
-LIBEXEC_SCRIPTS=	debug.sh hooks.sh safe_eval.sh
-LIBEXEC_SCRIPTSDIR=	/libexec
-LIBEXEC_SCRIPTSMODE=	755
-LIBEXEC_SCRIPTSPACKAGE=	rc
 
 SUBDIR+=	rc.d
 
