diff -Naur src.orig/Makefile.inc1 src/Makefile.inc1
--- src.orig/Makefile.inc1	2025-11-28 04:15:47.000000000 +0400
+++ src/Makefile.inc1	2025-12-27 12:25:14.596190000 +0400
@@ -1393,16 +1393,8 @@
 #
 # Check for missing UIDs/GIDs.
 #
-CHECK_UIDS=	auditdistd
-CHECK_GIDS=	audit
-CHECK_UIDS+=	ntpd
-CHECK_GIDS+=	ntpd
-CHECK_UIDS+=	proxy
-CHECK_GIDS+=	proxy authpf
-CHECK_UIDS+=	smmsp
-CHECK_GIDS+=	smmsp
-CHECK_UIDS+=	unbound
-CHECK_GIDS+=	unbound
+CHECK_UIDS=
+CHECK_GIDS=
 _installcheck_world: __installcheck_UGID
 __installcheck_UGID: .PHONY
 .for uid in ${CHECK_UIDS}
@@ -1447,10 +1439,6 @@
 	ln make mkdir mtree mv pwd_mkdb \
 	rm sed services_mkdb sh sort strip ${_sysctl} test time true uname wc
 
-.if ${MK_ZONEINFO} != "no"
-ITOOLS+=tzsetup
-.endif
-
 # Needed for share/man
 .if ${MK_MAN_UTILS} != "no"
 ITOOLS+=makewhatis
@@ -2768,11 +2756,6 @@
 _basic_bootstrap_tools+=sbin/md5
 .endif
 
-# tzsetup is needed as an install tool
-.if ${MK_ZONEINFO} != "no"
-_basic_bootstrap_tools+=usr.sbin/tzsetup
-.endif
-
 # certctl is needed as an install tool.  libcrypto is rather heavy, so we'll
 # build that alongside it only for platforms that don't expose headers for
 # OpenSSL, like macOS, or when building on releases with OpenSSL 1.x.
@@ -3430,10 +3413,8 @@
 cddl/lib/libdtrace__L: lib/libxo__L
 .endif
 # cddl/lib/libdtrace requires lib/libproc and lib/librtld_db
-_prebuild_libs+=	lib/libprocstat lib/libproc lib/librtld_db
+_prebuild_libs+=	lib/libprocstat
 lib/libprocstat__L: lib/libelf__L lib/libkvm__L lib/libutil__L
-lib/libproc__L: lib/libprocstat__L
-lib/librtld_db__L: lib/libprocstat__L
 
 .if ${MK_CRYPT} != "no"
 .if ${MK_OPENSSL} != "no"
@@ -3560,10 +3541,6 @@
 .if ${MK_OPENSSL} == "no"
 lib/libradius__L: lib/libmd__L
 .endif
-
-lib/libproc__L: \
-    ${_cddl_lib_libctf:D${_cddl_lib_libctf}__L} \
-    lib/libelf__L lib/librtld_db__L lib/libutil__L lib/libcxxrt__L
 
 .for _lib in ${_prereq_libs}
 ${_lib}__PL: .PHONY .MAKE
