diff -Naur src.orig/release/Makefile src/release/Makefile
--- src.orig/release/Makefile	2026-01-08 19:00:02.190827000 +0400
+++ src/release/Makefile	2026-01-10 11:49:25.204962000 +0400
@@ -235,33 +235,38 @@
 	    DESTDIR=${.OBJDIR}/${.TARGET} MK_AT=no \
 	    MK_INSTALLLIB=no MK_MAIL=no \
 	    ${_ALL_LIBCOMPATS:@v@MK_LIB$v=no@} \
+	    MK_INCLUDES=no MK_PF=no \
+	    MK_INSTALLLIB=no MK_KDUMP=no \
+	    MK_LDNS_UTILS=no MK_NETCAT=no \
+	    MK_CUSE=no MK_MAKE=no MK_MAN=no \
+	    MK_OPENSSH=no MK_PKGBOOTSTRAP=no \
 	    MK_TOOLCHAIN=no \
 	    MK_RESCUE=no MK_DICT=no \
 	    MK_KERNEL_SYMBOLS=no MK_TESTS=no MK_DEBUG_FILES=no)
 .if !defined(NOPKGBASE) || empty(NOPKGBASE)
 # Create offline pkgbase repo on release media
-	mkdir -p ${.TARGET}/usr/freebsd-packages/repos/
+	mkdir -p ${.TARGET}/usr/packages/repos/
 	${.CURDIR}/scripts/pkgbase-stage.lua disc \
 	    ${.OBJDIR}/pkgbase-repo-dir \
-	    ${.TARGET}/usr/freebsd-packages/offline \
+	    ${.TARGET}/usr/packages/offline \
 	    "${_ALL_libcompats}" ${PKG_ABI} ${.OBJDIR}/pkgdb-disc1
 	cp ${.CURDIR}/scripts/FreeBSD-base-offline.conf \
-		${.TARGET}/usr/freebsd-packages/repos/
-	mtree -c -p ${.TARGET}/usr/freebsd-packages | \
+		${.TARGET}/usr/packages/repos/
+	mtree -c -p ${.TARGET}/usr/packages | \
 	    mtree -C -k type,mode,link,size | \
-	    sed 's|^\.|./usr/freebsd-packages|g' >> ${.TARGET}/METALOG
+	    sed 's|^\.|./usr/packages|g' >> ${.TARGET}/METALOG
 .endif
 .if !defined(NODISTSETS) || empty(NODISTSETS)
 	# Copy MANIFEST to provide legacy dist checksums in both modes
-	mkdir -p ${.TARGET}/usr/freebsd-dist
-	echo "./usr/freebsd-dist type=dir uname=root gname=wheel mode=0755" >> ${.TARGET}/METALOG
-	cp MANIFEST ${.TARGET}/usr/freebsd-dist
-	echo "./usr/freebsd-dist/MANIFEST type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG
+	mkdir -p ${.TARGET}/usr/distsets
+	echo "./usr/distsets type=dir uname=root gname=wheel mode=0755" >> ${.TARGET}/METALOG
+	cp MANIFEST ${.TARGET}/usr/distsets
+	echo "./usr/distsets/MANIFEST type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG
 .if defined(NOPKGBASE) && !empty(NOPKGBASE)
 # Copy distfiles
 	for dist in $$(ls *.txz | grep -v container | grep -vE -- '(${base ${_ALL_libcompats}:L:ts|})-dbg'); do \
-	    cp $${dist} ${.TARGET}/usr/freebsd-dist; \
-	    echo "./usr/freebsd-dist/$${dist} type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG; \
+	    cp $${dist} ${.TARGET}/usr/distsets; \
+	    echo "./usr/distsets/$${dist} type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG; \
 	done
 .endif
 .endif
@@ -277,19 +282,45 @@
 	${PKG_CLEAN} || true
 .endif
 # Set up installation environment
-	ln -fs /tmp/bsdinstall_etc/resolv.conf ${.TARGET}/etc/resolv.conf
-	echo sendmail_enable=\"NONE\" > ${.TARGET}/etc/rc.conf
-	echo hostid_enable=\"NO\" >> ${.TARGET}/etc/rc.conf
-	echo vfs.mountroot.timeout=\"10\" >> ${.TARGET}/boot/loader.conf
+	echo autoboot_delay=\"4\" > ${.TARGET}/boot/loader.conf
+	echo bootenv_autolist=\"NO\" >> ${.TARGET}/boot/loader.conf
+	echo entropy_cache_load=\"NO\" >> ${.TARGET}/boot/loader.conf
 	echo kernels_autodetect=\"NO\" >> ${.TARGET}/boot/loader.conf
-	echo loader_brand=\"install\" >> ${.TARGET}/boot/loader.conf
-	echo loader_menu_multi_user_prompt=\"Installer\" >> ${.TARGET}/boot/loader.conf
-	cp ${.CURDIR}/rc.local ${.TARGET}/etc
-	echo "./etc/resolv.conf type=link uname=root gname=wheel mode=0644 link=/tmp/bsdinstall_etc/resolv.conf" >> ${.TARGET}/METALOG
-	echo "./etc/rc.conf type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG
-	echo "./etc/sysctl.conf type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG
-	echo "./boot/loader.conf type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG
-	echo "./etc/rc.local type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG
+	echo vbe_max_resolution=\"640x480\" >> ${.TARGET}/boot/loader.conf
+	echo loader_brand_y=\"2\" >> ${.TARGET}/boot/loader.conf
+	echo loader_logo_x=\"51\" >> ${.TARGET}/boot/loader.conf
+	echo loader_logo_y=\"14\" >> ${.TARGET}/boot/loader.conf
+	echo loader_menu_title=\"LiveCD\" >> ${.TARGET}/boot/loader.conf
+	echo vfs.mountroot.timeout=\"10\" >> ${.TARGET}/boot/loader.conf
+	echo "console none unknown off secure" > ${.TARGET}/etc/ttys
+	echo "ttyv0 \"/usr/libexec/getty al.Pc\" xterm onifexists secure" >> ${.TARGET}/etc/ttys
+	echo "ttyv1 \"/usr/libexec/getty Pc\"    xterm onifexists secure" >> ${.TARGET}/etc/ttys
+	echo "ttyv2 \"/usr/libexec/getty Pc\"    xterm onifexists secure" >> ${.TARGET}/etc/ttys
+	pw -R ${.TARGET} usermod -n root -w none
+	cp ${.CURDIR}/rc ${.CURDIR}/rc.shutdown ${.CURDIR}/sysctl.conf ${.TARGET}/etc
+	cp ${.CURDIR}/corebsd.install ${.TARGET}/root
+	mv ${.TARGET}/usr/share/zoneinfo/Etc/UTC ${.TARGET}/etc/localtime
+	sed -E -i '' \
+		-e '/\/boot\/(loader\.conf\.d|modules|zfs )/d' \
+		-e '/\/etc\/(devd|dhclient|etcupdate|libmap|moused|rc|(new)?syslog)\.conf/d' \
+		-e '/\/etc\/(cron(\.d|tab)|devd|fbtab|login.access|pkg|rc\.(d|resume|suspend)|syslog\.d|ss(h|l))/d' \
+		-e '/\/etc\/mtree\/BSD\.(include|root|usr)\.dist/d' \
+		-e '/\/etc\/pam\.d\/(cron|sshd|xdm)/d' \
+		-e '/\/usr\/(include|libdata|local|obj|share\/openssl|src)/d' \
+		-e '/\/usr\/libexec\/(flua|ulog-helper)/d' \
+		-e '/\/usr\/lib\/(.+\.(a|o)|clang|flua|libxo)/d' \
+		-e '/\/bin\/(crontab|iconv)/d' \
+		-e '/\/sbin\/(cron|devd|etcupdate|mixer|moused|(new)?syslogd?|nologin|powerd)/d' \
+		-e '/\/share\/(certs|keys|man|mk|moused|pkg|skel|zoneinfo)/d' \
+		-e '/\/var\/(cache|cron|db|empty|log|run|tmp)/d' \
+		-e '/\.\/tmp/d' \
+	${.TARGET}/METALOG
+	echo "./etc/localtime type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG
+	echo "./etc/resolv.conf type=link uname=root gname=wheel mode=0755 link=/var/run/resolv.conf" >> ${.TARGET}/METALOG
+	echo "./tmp type=link uname=root gname=wheel mode=0755 link=/var/tmp" >> ${.TARGET}/METALOG
+	echo "./root/corebsd.install type=file uname=root gname=wheel mode=0555" >> ${.TARGET}/METALOG
+	sed -E 's/[[:blank:]]tags=package=[[:graph:]]+//' ${.TARGET}/METALOG | sort | uniq > ${.TARGET}/METALOG.tmp
+	mv ${.TARGET}/METALOG.tmp ${.TARGET}/METALOG
 	touch ${.TARGET}

 bootonly:
