diff -Naur src.orig/release/Makefile src/release/Makefile
--- src.orig/release/Makefile	2025-06-06 04:04:20.000000000 +0400
+++ src/release/Makefile	2025-10-01 16:09:15.190083000 +0400
@@ -180,16 +180,19 @@
 	mkdir -p ${.TARGET}
 	( cd ${WORLDDIR} && ${IMAKE} installworld installkernel distribution \
 	    DESTDIR=${.OBJDIR}/${.TARGET} MK_AT=no \
-	    MK_INSTALLLIB=no MK_MAIL=no \
+	    MK_INCLUDES=no MK_PF=no \
 	    ${_ALL_LIBCOMPATS:@v@MK_LIB$v=no@} \
+	    MK_INSTALLLIB=no MK_KDUMP=no \
+	    MK_LDNS_UTILS=no MK_MAKE=no \
+	    MK_MAN=no MK_NETCAT=no \
+	    MK_GSSAPI=no MK_KERBEROS=no \
+	    MK_OPENSSH=no MK_PKGBOOTSTRAP=no \
 	    MK_TOOLCHAIN=no MK_PROFILE=no \
-	    MK_RESCUE=no MK_DICT=no \
-	    MK_KERNEL_SYMBOLS=no MK_TESTS=no MK_DEBUG_FILES=no \
 	    -DDB_FROM_SRC )
 # Copy distfiles
-	mkdir -p ${.TARGET}/usr/freebsd-dist
+	mkdir -p ${.TARGET}/usr/corebsd
 	for dist in MANIFEST $$(ls *.txz | grep -v container | grep -vE -- '(${base ${_ALL_libcompats}:L:ts|})-dbg'); \
-	    do cp $${dist} ${.TARGET}/usr/freebsd-dist; \
+	    do cp $${dist} ${.TARGET}/usr/corebsd; \
 	done
 .if defined(NO_ROOT)
 	echo "./usr/freebsd-dist type=dir uname=root gname=wheel mode=0755" >> ${.TARGET}/METALOG
@@ -202,13 +205,44 @@
 	${PKG_INSTALL} wifi-firmware-kmod-release || true
 .endif
 # Set up installation environment
-	ln -fs /tmp/bsdinstall_etc/resolv.conf ${.TARGET}/etc/resolv.conf
-	echo sendmail_enable=\"NONE\" > ${.TARGET}/etc/rc.conf
-	echo hostid_enable=\"NO\" >> ${.TARGET}/etc/rc.conf
-	echo vfs.mountroot.timeout=\"10\" >> ${.TARGET}/boot/loader.conf
+	echo autoboot_delay=\"4\" > ${.TARGET}/boot/loader.conf
+	echo bootenv_autolist=\"NO\" >> ${.TARGET}/boot/loader.conf
+	echo entropy_cache_load=\"NO\" >> ${.TARGET}/boot/loader.conf
 	echo kernels_autodetect=\"NO\" >> ${.TARGET}/boot/loader.conf
-	echo loader_menu_multi_user_prompt=\"Installer\" >> ${.TARGET}/boot/loader.conf
-	cp ${.CURDIR}/rc.local ${.TARGET}/etc
+	echo vbe_max_resolution=\"640x480\" >> ${.TARGET}/boot/loader.conf
+	echo loader_brand_y=\"2\" >> ${.TARGET}/boot/loader.conf
+	echo loader_logo_x=\"51\" >> ${.TARGET}/boot/loader.conf
+	echo loader_logo_y=\"8\" >> ${.TARGET}/boot/loader.conf
+	echo loader_menu_title=\"LiveCD\" >> ${.TARGET}/boot/loader.conf
+	echo vfs.mountroot.timeout=\"10\" >> ${.TARGET}/boot/loader.conf
+	echo "console none unknown off secure" > ${.TARGET}/etc/ttys
+	for i in 0 1 2 3; do \
+	    echo "ttyv$${i} \"/usr/libexec/getty autologin\" xterm onifexists secure" >> ${.TARGET}/etc/ttys; \
+	done
+	chflags -R noschg ${.TARGET}/
+	cp ${.CURDIR}/rc ${.TARGET}/etc
+	cp ${.CURDIR}/rc.shutdown ${.TARGET}/etc
+	cp ${.CURDIR}/sysctl.conf ${.TARGET}/etc
+	cp ${.CURDIR}/corebsd.install ${.TARGET}/root
+	mv ${.TARGET}/usr/share/zoneinfo/Etc/UTC ${.TARGET}/etc/localtime
+	find ${.TARGET}/boot ${.TARGET}/etc ${.TARGET}/usr ${.TARGET}/var -mindepth 1 -type d -empty -delete
+	rm -rf \
+	    ${.TARGET}/boot/fonts/!(8x16.fnt.gz) \
+	    ${.TARGET}/etc/{crontab,devd,fbtab,login.access,networks,ssl} \
+	    ${.TARGET}/etc/{devd,dhclient,etcupdate,libmap,rc,{,new}syslog}.conf \
+	    ${.TARGET}/etc/mtree/!(BSD.var.dist) \
+	    ${.TARGET}/etc/pam.d/{cron,sshd,xdm} \
+	    ${.TARGET}/etc/rc.{d,suspend} \
+	    ${.TARGET}/sbin/{devd,nologin} \
+	    ${.TARGET}/usr/lib/libxo \
+	    ${.TARGET}/usr/libdata \
+	    ${.TARGET}/usr/libexec/flua \
+	    ${.TARGET}/usr/bin/{crontab,iconv} \
+	    ${.TARGET}/usr/sbin/{cron,etcupdate,mixer,moused,newsyslog,nologin,powerd,syslogd} \
+	    ${.TARGET}/usr/share/{certs,keys,man,mk,skel,zoneinfo} \
+	    ${.TARGET}/{sys,tmp}
+	ln -sf /var/tmp ${.TARGET}/tmp
+	ln -sf /var/run/resolv.conf ${.TARGET}/etc/resolv.conf
 .if defined(NO_ROOT)
 	echo "./etc/resolv.conf type=link uname=root gname=wheel mode=0644 link=/tmp/bsdinstall_etc/resolv.conf" >> ${.TARGET}/METALOG
 	echo "./etc/rc.conf type=file uname=root gname=wheel mode=0644" >> ${.TARGET}/METALOG
