diff -Naur src.orig/sbin/devd/devmatch.conf src/sbin/devd/devmatch.conf
--- src.orig/sbin/devd/devmatch.conf	2025-06-06 00:04:20.000000000 +0000
+++ src/sbin/devd/devmatch.conf	2025-08-15 15:01:19.153047000 +0000
@@ -1,19 +1,8 @@
-#
-#
-
-#
-# Example devd configuration file for automatically
-# loading what modules we can based on nomatch
-# events.
-#
-# Generic NOMATCH event
 nomatch 100 {
-	action "service devmatch quietstart '?'$_";
+	action "service devmatch '?'$_";
 };
 
 # Add the following to devd.conf to prevent this from running:
-# 	nomatch 1000 {
-#		action "true";
-#	};
-# it replaces the generic event with one of higher priority that
-# does nothing. You can also set 'devmatch_enable=NO' in /etc/rc.conf
+# nomatch 1000 {
+#	action "true";
+# };
