diff -Naur src.orig/sbin/Makefile src/sbin/Makefile
--- src.orig/sbin/Makefile	2025-06-06 00:04:20.000000000 +0000
+++ src/sbin/Makefile	2025-08-02 10:40:05.430416000 +0000
@@ -4,30 +4,13 @@
 
 SUBDIR=adjkerntz \
 	camcontrol \
-	clri \
-	comcontrol \
-	conscontrol \
-	ddb \
 	devd \
 	devfs \
 	devmatch \
 	dhclient \
 	dmesg \
-	dump \
-	dumpfs \
-	dumpon \
-	etherswitchcfg \
-	ffsinfo \
-	fsck \
-	fsck_ffs \
 	fsck_msdosfs \
-	fsdb \
-	fsirand \
-	gbde \
 	geom \
-	ggate \
-	growfs \
-	gvinum \
 	ifconfig \
 	init \
 	kldconfig \
@@ -37,35 +20,23 @@
 	ldconfig \
 	md5 \
 	mdconfig \
-	mdmfs \
 	mknod \
-	mksnap_ffs \
 	mount \
 	mount_cd9660 \
 	mount_fusefs \
 	mount_msdosfs \
-	mount_nfs \
 	mount_nullfs \
 	mount_udf \
 	mount_unionfs \
-	newfs \
 	newfs_msdos \
-	nfsiod \
-	nos-tun \
 	nvmecontrol \
-	pfilctl \
-	rcorder \
 	reboot \
 	recoverdisk \
 	resolvconf \
-	restore \
 	route \
-	savecore \
-	setkey \
 	shutdown \
 	swapon \
 	sysctl \
-	tunefs \
 	umount
 
 .if ${MK_INET} != "no" || ${MK_INET6} != "no"
@@ -78,8 +49,6 @@
 SUBDIR.${MK_INET6}+=	rtsol
 SUBDIR.${MK_IPFILTER}+=	ipf
 SUBDIR.${MK_IPFW}+=	ipfw
-SUBDIR.${MK_IPFW}+=	natd
-SUBDIR.${MK_OPENSSL}+=	decryptcore
 SUBDIR.${MK_PF}+=	pfctl
 SUBDIR.${MK_PF}+=	pflogd
 SUBDIR.${MK_QUOTAS}+=	quotacheck
