diff -Naur src.orig/sbin/devd/Makefile src/sbin/devd/Makefile
--- src.orig/sbin/devd/Makefile	2025-11-28 04:15:53.000000000 +0400
+++ src/sbin/devd/Makefile	2026-01-19 12:56:24.019232000 +0400
@@ -6,9 +6,6 @@
 CONFS=	devd.conf
 DEVD=	devmatch.conf
 DEVDDIR=	/etc/devd
-.if ${MK_ACPI} != "no"
-DEVD+=	asus.conf
-.endif
 
 .if ${MK_AUTOFS} != "no"
 CONFGROUPS+=		AUTOFS
@@ -27,11 +24,6 @@
 POWERPROFILE+=		power_profile.conf
 POWERPROFILEPACKAGE=	acpi
 
-CONFGROUPS+=		CONSOLE
-CONSOLEDIR=		${DEVDDIR}
-CONSOLE+=		moused.conf syscons.conf
-CONSOLEPACKAGE=		console-tools
-
 CONFGROUPS+=		SND
 SNDDIR=			${DEVDDIR}
 SND=			snd.conf
@@ -55,10 +47,6 @@
 NVMEDIR=		${DEVDDIR}
 NVME+=			nvmf.conf
 NVMEPACKAGE=		nvme-tools
-
-.if ${MK_USB} != "no"
-DEVD+=	uath.conf ulpt.conf
-.endif
 
 .if ${MACHINE_ARCH} == "powerpc"
 DEVD+=	apple.conf
