diff -Naur src.orig/secure/lib/libcrypto/Makefile src/secure/lib/libcrypto/Makefile
--- src.orig/secure/lib/libcrypto/Makefile	2025-06-06 03:04:20.000000000 +0300
+++ src/secure/lib/libcrypto/Makefile	2025-06-17 11:47:31.867438000 +0300
@@ -8,14 +8,13 @@
 .include <src.opts.mk>
 
 PACKAGE=	openssl-lib
-LIB=		crypto
+SHLIB=		crypto
 SHLIB_MAJOR=	30
 VERSION_MAP=	${.CURDIR}/Version.map
 
 NO_LINT=
 PCFILES=	libcrypto.pc
 
-.include "Makefile.man"
 .include "Makefile.inc"
 
 SRCS=	asn1_dsa.c bsearch.c cpt_err.c context.c core_algorithm.c core_fetch.c core_namemap.c cpuid.c cryptlib.c ctype.c
