diff -Naur src.orig/share/man/man8/Makefile src/share/man/man8/Makefile
--- src.orig/share/man/man8/Makefile	2025-06-06 00:04:21.000000000 +0000
+++ src/share/man/man8/Makefile	2025-08-10 14:04:57.938511000 +0000
@@ -7,23 +7,11 @@
 	crash.8 \
 	diskless.8 \
 	intro.8 \
-	nanobsd.8 \
-	rc.8 \
-	rc.subr.8 \
 	rescue.8 \
 	${_uefi.8} \
 
 MLINKS= \
-	beinstall.8 beinstall.sh.8 \
-	nanobsd.8 nanobsd.sh.8 \
-	rc.8 rc.d.8 \
-	rc.8 rc.firewall.8 \
-	rc.8 rc.local.8 \
-	rc.8 rc.network.8 \
-	rc.8 rc.pccard.8 \
-	rc.8 rc.resume.8 \
-	rc.8 rc.serial.8 \
-	rc.8 rc.shutdown.8
+	beinstall.8 beinstall.sh.8
 
 .if ${MK_NIS} != "no"
 MAN+=	yp.8
