diff -Naur src.orig/stand/defaults/Makefile src/stand/defaults/Makefile
--- src.orig/stand/defaults/Makefile	2025-11-28 04:15:54.000000000 +0400
+++ src/stand/defaults/Makefile	2025-12-27 16:34:53.777850000 +0400
@@ -1,8 +1,10 @@
 .include <bsd.init.mk>
 
-FILES+=	loader.conf
+FILES+=	loader.conf loader_local.conf
 MAN+=	loader.conf.5
 
-FILESDIR_loader.conf=  /boot/defaults
+FILESDIR=			/boot
+FILESNAME_loader_local.conf=	loader.conf
+FILESDIR_loader.conf=		/boot/defaults
 
 .include <bsd.prog.mk>
