diff -Naur src.orig/stand/lua/Makefile src/stand/lua/Makefile
--- src.orig/stand/lua/Makefile	2025-11-28 04:15:54.000000000 +0400
+++ src/stand/lua/Makefile	2025-12-28 14:32:35.995985000 +0400
@@ -21,12 +21,7 @@
 	drawer.lua \
 	hook.lua \
 	loader.lua \
-	gfx-beastie.lua \
-	gfx-beastiebw.lua \
-	gfx-fbsdbw.lua \
-	gfx-install.lua \
 	gfx-orb.lua \
-	gfx-orbbw.lua \
 	menu.lua \
 	password.lua \
 	screen.lua
