diff -Naur src.orig/stand/Makefile.amd64 src/stand/Makefile.amd64
--- src.orig/stand/Makefile.amd64	2025-11-28 04:15:54.000000000 +0400
+++ src/stand/Makefile.amd64	2025-12-28 14:36:07.366592000 +0400
@@ -1,6 +1,4 @@
-S.yes+=		userboot
 S.yes+=		i386
 
-SUBDIR_DEPEND_userboot+=	${INTERP_DEPENDS}
 # These won't get tacked on in an amd64 build
 SUBDIR_DEPEND_i386+=	${LIB32DEPENDS} ${INTERP_DEPENDS}
