diff -Naur src.orig/sys/conf/newvers.sh src/sys/conf/newvers.sh
--- src.orig/sys/conf/newvers.sh	2025-11-28 04:15:54.000000000 +0400
+++ src/sys/conf/newvers.sh	2025-12-28 14:41:32.023551000 +0400
@@ -116,7 +116,7 @@
 while getopts cd:rRvV: opt; do
 	case "$opt" in
 	c)
-		echo "$COPYRIGHT"
+		echo ""
 		exit 0
 		;;
 	d)
@@ -299,11 +299,10 @@
 	VERSTR="${VERINFO}\\n"
 else
 	VERINFO="${VERSION} #${v}${svn}${git}${gitup}${hg}: ${t}"
-	VERSTR="${VERINFO}\\n    ${u}@${h}:${d}\\n"
+	VERSTR="${VERINFO}\\n    ${u}@${h}\\n"
 fi
 
 vers_content_new=$(cat << EOF
-$COPYRIGHT
 /*
  * The SCCS stuff is a marker that by convention identifies the kernel.  While
  * the convention originated with SCCS, the current use is more generic and is
