diff -Naur src.orig/usr.sbin/Makefile.amd64 src/usr.sbin/Makefile.amd64
--- src.orig/usr.sbin/Makefile.amd64	2025-06-06 00:04:27.000000000 +0000
+++ src/usr.sbin/Makefile.amd64	2025-06-26 05:41:05.194444000 +0000
@@ -12,12 +12,4 @@
 SUBDIR+=	bhyvectl
 SUBDIR+=	bhyveload
 .endif
-SUBDIR+=	boot0cfg
-.if ${MK_TOOLCHAIN} != "no"
-SUBDIR+=	btxld
-.endif
-SUBDIR+=	cpucontrol
-SUBDIR+=	lptcontrol
-SUBDIR+=	mptable
-SUBDIR+=	spkrtest
 SUBDIR+=	zzz
