diff -Naur src.orig/usr.bin/clang/Makefile src/usr.bin/clang/Makefile
--- src.orig/usr.bin/clang/Makefile	2026-01-08 18:59:31.868177000 +0400
+++ src/usr.bin/clang/Makefile	2026-01-10 14:57:30.070375000 +0400
@@ -12,7 +12,7 @@
 # LLVM binutils are needed to support features such as LTO, so we build them
 # by default if clang is enabled. If MK_LLVM_BINUTILS is set, we also use them
 # as the default binutils (ar,nm,addr2line, etc.).
-.if ${MK_CLANG} != "no" || ${MK_LLVM_BINUTILS} != "no"
+.if ${MK_CLANG} != "no" && ${MK_LLVM_BINUTILS} != "no"
 SUBDIR+=	llvm-ar
 SUBDIR+=	llvm-nm
 SUBDIR+=	llvm-objcopy
