diff -Naur src.orig/usr.sbin/wpa/wpa_supplicant/Makefile src/usr.sbin/wpa/wpa_supplicant/Makefile
--- src.orig/usr.sbin/wpa/wpa_supplicant/Makefile	2026-06-12 10:36:48.000000000 +0400
+++ src/usr.sbin/wpa/wpa_supplicant/Makefile	2026-06-16 19:08:31.346141000 +0400
@@ -46,6 +46,9 @@
 FILES=	wpa_supplicant.conf
 .endif
 
+CONFS=	wpa_supplicant.conf
+CONFSMODE=	640
+
 CFLAGS+=-DCONFIG_BACKEND_FILE
 #CFLAGS+= -g
 LIBADD+=pcap util wpadrivers wpaeapol_supp \
